store/sqlstore/store.go

```diff
@@ -323,6 +324,25 @@ func (ss *SqlStore) DriverName() string {
 	return *ss.settings.DriverName
 }
 
+// IsBinaryParamEnabled returns whether the data source uses binary_parameters
+// when using Postgres
+func (ss *SqlStore) IsBinaryParamEnabled() (bool, error) {
+	if ss.DriverName() != model.DatabaseDriverPostgres {
+		return false, nil
+	}
+
+	url, err := url.Parse(*ss.settings.DataSource)
```

Comment:
Should we compute this every time? Maybe we could just set a private boolean field?

Reply:
Excellent idea